backend/api/student/cohorts.py

```
@@ -92,10 +96,14 @@ async def get_cohort_simulations(
     if not enrollment:
         raise HTTPException(status_code=403, detail="Not enrolled in this cohort")
     
-    # Get simulations assigned to this cohort
+    # Get simulations assigned to this cohort (only published/active simulations)
     simulations_query = db.query(CohortSimulation, Scenario).join(
         Scenario, CohortSimulation.simulation_id == Scenario.id
-    ).filter(CohortSimulation.cohort_id == cohort.id)
+    ).filter(
+        CohortSimulation.cohort_id == cohort.id,
+        Scenario.is_draft == False,  # Only show published simulations to students
```

Ruff (0.14.0)

Avoid equality comparisons to `False`; use `not Scenario.is_draft:` for false checks

Replace with `not Scenario.is_draft`

(E712)